test/bench.cpp

```diff
@@ -4,87 +4,370 @@
 #include <string>
 #include <vector>
 
+#include <sstream>
+
+#include <stdio.h>
+#include <stdbool.h>
+
+#include <SDL.h>
+#include <SDL_ttf.h>
+#include <cstdio>
+#include <ctype.h>
+
+#define MAX(X, Y) (((X) > (Y)) ? (X) : (Y))
+#define MIN(X, Y) (((X) < (Y)) ? (X) : (Y))
+
+// Define screen dimensions
+#define SCREEN_WIDTH    1200
+#define SCREEN_HEIGHT   600
+
+// Define params of input squares
+#define INPUT_WIDTH   60
+#define INPUT_HEIGHT   45
```

Comment:
Non-configurable integer constants must be enumerations, e.g.
```suggestion
enum {
    INPUT_WIDTH = 60,
    INPUT_HEIGHT = 45
}
```